pkg/azurePipelines/azurePipelines.go

```diff
@@ -83,6 +84,36 @@ func (p *AzurePipelines) loadConfig(deployType string) (*config.DraftConfig, err
 	return &draftConfig, nil
 }
 
+func (p *AzurePipelines) overrideFilename(draftConfig *config.DraftConfig, srcDir string) error {
+	if draftConfig.FileNameOverrideMap == nil {
+		draftConfig.FileNameOverrideMap = make(map[string]string)
+	}
+	pipelineVar, err := draftConfig.GetVariable(pipelineNameVar)
+	if err != nil {
+		return fmt.Errorf("error getting pipeline name variable: %w", err)
+	}
+
+	files, err := fs.ReadDir(p.pipelineTemplates, srcDir)
+	if err != nil {
+		return fmt.Errorf("error reading source directory: %w", err)
+	}
+
+	for _, f := range files {
+		if f.Name() == configFileName {
+			continue
+		}
+		if f.IsDir() {
```

Comment:
this looks a lot like [WalkDir](https://pkg.go.dev/path/filepath#WalkDir) it might be work just using the built in implementation

Reply:
Thanks, changed the code to use WalkDir, it's a lot cleaner!